app/classes/query/modules/lookup_names.rb

```diff
@@ -191,7 +223,7 @@ def minimal_name_data(name)
       end
 
       def minimal_name_columns
-        "id, correct_spelling_id, synonym_id, text_name"
+        [:id, :correct_spelling_id, :synonym_id, :text_name].freeze
       end
```

Comment:
QUESTION: Would it be better to use a constant instead of a method?

Reply:
No opinion.